src/orchestratord/src/controller/materialize.rs

```diff
@@ -41,6 +41,8 @@ pub struct Args {
     #[clap(flatten)]
     aws_info: AwsInfo,
 
+    #[clap(long)]
+    ephemeral_volume_class: Option<String>,
```

Comment:
Can we keep the names of variables and fields consistent? Everywhere else you refer to this as `ephemeral_storage_class`, which seems more explanatory.

Reply:
i was trying to stay consistent with the cli flag names we're already using in environmentd and clusterd - i've cleaned things up a bit more to try to standardize on that naming.